src/test/kotlin/dev/stalla/builder/validating/ValidatingPodcastindexLocationBuilderTest.kt

```diff
@@ -0,0 +1,89 @@
+package dev.stalla.builder.validating
+
+import assertk.all
+import assertk.assertAll
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isFalse
+import assertk.assertions.isNotNull
+import assertk.assertions.isNull
+import assertk.assertions.isTrue
+import assertk.assertions.prop
+import dev.stalla.builder.PodcastindexLocationBuilder
+import dev.stalla.model.aPodcastindexGeographicLocation
+import dev.stalla.model.aPodcastindexOpenStreetMapElement
+import dev.stalla.model.anEpisodePodcastindexLocation
+import dev.stalla.model.podcastindex.PodcastindexLocation
+import org.junit.jupiter.api.Test
+
+internal class ValidatingPodcastindexLocationBuilderTest {
+
+    @Test
+    internal fun `should not build an Podcastindex Location with when all the fields are missing`() {
```

Comment:
```suggestion
    internal fun `should not build a Podcastindex Location when all the fields are missing`() {
```

Reply:
Technically, this test is redundant, given the name field is mandatory and as far as iI can tell it's the only mandatory field.

Reply:
True, but we have this test also for other builders when the corresponding model has only one mandatory field (`Guid`, `Enclosure`), so I've added them for consistency.